### ISSUE TITLE
Docs: link to documentation for creating custom templates is invalid

### ISSUE THREAD

**Issue by twprzybysz** [SwiftGen/SwiftGen#724]
The following link in topmost README.md is pointing to 404 resource - https://github.com/SwiftGen/SwiftGen/blob/stable/Documentation/Creating-your-templates.md
```
For more information about how to create your own templates, see the dedicated documentation.
```

**Comment by djbe** [comment]
Thanks for reporting this issue!

A fix has been implemented in #729 (commit cc288ad6c73a87835b0e457b0be690b6e3943005). If you notice any other issues, please let us know.

**Comment by djbe** [comment]
Should be fixed in SwiftGen 6.2.1 🎉
